src/MacroExpander.ts

```diff
@@ -30,7 +28,7 @@ export const implicitCommands = {
 export default class MacroExpander implements MacroContextInterface {
     settings: Settings;
     expansionCount: number;
-    lexer: Lexer;
+    lexer!: Lexer;
```

Comment:
@kevinbarabash `!` is a [non-null assertion operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator), since `feed`, which sets the `lexer`, is always called.